docs/content/QuickStart.fsx

```diff
@@ -10,7 +10,20 @@ The fastest way to understand how FsCheck works is by writing some *properties*
 test, or a generative test - and run them using the built-in test runner. Later on, we'll describe how they can be integrated
 with existing test frameworks like NUnit, xUnit.NET or MsTest.
 
-First install FsCheck, open an fsx file and start with:*)
+## Installation
+First install FsCheck. Make sure your project(s) has target `.NET 4.5` or greater.
+
+If you installed `FsCheck.NUnit` via Paket, make sure you have added these references to your project
+([Paket doesn't run any powershell scripts](https://fsprojects.github.io/Paket/faq.html#Does-Paket-run-install-ps1-scripts)
+that usually do this for you):
+
+```
+packages/NUnit.Runners/tools/lib/nunit.core.dll
+packages/NUnit.Runners/tools/lib/nunit.core.interfaces.dll
+```
+
+## Usage
+Open an fsx file and start with:*)
```

Comment:
Don't feel like this belongs here either, the quickstart uses vanilla FsCheck with a shallow reference to xUnit integration at the end. It would be better to extend the xunit section somewhat to basically merely acknowledge the existence of FsCheck.NUnit, but just put a link to the more NUnit section I mentioned earlier.